### ISSUE TITLE
Initial release checklist (v2)

### ISSUE THREAD

**Issue by annybs** [edge/account#65]
Supersedes #22 

These are remaining issues to work on between now (Friday 10th June) and prospective EOM launch, in approximate order of priority.

## Reviews
- #74 
  - pending review
- #75
  - pending review

## Major components

- #49 
  - IIRC from recent Discord chatter, logged-in index should redirect to server list, with a full dashboard deferred to a future release
- #59 
- #15 
  - Waiting on https://github.com/edge/account-api/issues/37

## Minor components

- #28
- #44
- #50
- #51 
- #54
- #56 
- #62 
- #63 
- #68
- #73  

## Nice to have

These _should_ be completed for initial release, but are not highest-priority.

- #43 
- #55 

## Post-launch

- #42
  - @willgarrett64 "It's prevented, but not super smooth"
- #46 
  - Waiting on https://github.com/edge/account-api/issues/31
- #48 
  - We do not currently have a search API - only a paginating API. Can we get away with disabling the search bar at launch and adding it back in later?
- #58 
  - Waiting on Account API (no issue created yet)
- #72
